libs/features/pages/home/src/lib/main/home/home-page.component.ts

```diff
@@ -1,8 +1,52 @@
 import { Component } from '@angular/core';
+import { SliderButtonData } from '@elewa-website/models/schema/ui/buttons';
 
 @Component({
   selector: 'elewa-website-home',
   templateUrl: './home-page.component.html',
   styleUrls: ['./home-page.component.scss'],
 })
-export class HomePageComponent {}
+export class HomePageComponent {
+  sliderButtonData: SliderButtonData = {
+
+    text: "Unlock your Potential",
+
+    bgColour: "black", 
+    color: "white",
+    borderColor: "",
+    iconColor: "black",
+    iconBackgroundColor: "white",
+  
+    hoverBgColour: "white",
+    hoverColor: "black",
+    hoverBorderColor: "",
+    hoverIconColor: "white",
+    hoverIconBackgroundColor: "black"
+
+  }
```

Comment:
No need to define any of this here.. Any component that needs this button will import it.

Reply:
this was for my testing I wasnt supposed to push the code and I have removed the data